homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -299,22 +316,34 @@
     ATTR_TARGET_HUMIDITY: "target_humidity",
     ATTR_LED_BRIGHTNESS: "led_brightness",
     ATTR_USE_TIME: "use_time",
-    ATTR_HARDWARE_VERSION: "hardware_version",
 }
 
 AVAILABLE_ATTRIBUTES_AIRHUMIDIFIER = {
     **AVAILABLE_ATTRIBUTES_AIRHUMIDIFIER_COMMON,
+    ATTR_HARDWARE_VERSION: "hardware_version",
     ATTR_TRANS_LEVEL: "trans_level",
     ATTR_BUTTON_PRESSED: "button_pressed",
 }
 
 AVAILABLE_ATTRIBUTES_AIRHUMIDIFIER_CA_AND_CB = {
     **AVAILABLE_ATTRIBUTES_AIRHUMIDIFIER_COMMON,
+    ATTR_HARDWARE_VERSION: "hardware_version",
     ATTR_MOTOR_SPEED: "motor_speed",
     ATTR_DEPTH: "depth",
     ATTR_DRY: "dry",
 }
 
+AVAILABLE_ATTRIBUTES_AIRHUMIDIFIER_CA4 = {
+    **AVAILABLE_ATTRIBUTES_AIRHUMIDIFIER_COMMON,
+    ATTR_ACTUAL_MOTOR_SPEED: "actual_speed",
+    ATTR_BUTTON_PRESSED: "button_pressed",
+    ATTR_DRY: "dry",
+    ATTR_FAHRENHEIT: "fahrenheit",
+    ATTR_MOTOR_SPEED: "motor_speed",
+    ATTR_POWER_TIME: "power_time",
```

Comment:
What does power time measure?

Reply:
Time the hum was "powered" the is property from API of the hum (lib)

Reply:
This should be a separate sensor entity with device class timestamp.
https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes